database/factories/QuestionAttachmentFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Question;
+use App\QuestionAttachment;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Faker\Generator as Faker;
+
+$factory->define(QuestionAttachment::class, function (Faker $faker) {
+    $type = $faker->randomElement(['image', 'audio', 'video']);
+    //$type = 'image';
+    return [
+        'path' => function() use ($type) {
+            if ($type == 'image'){
```

Comment:
consider cleaning up these if-else ladder.

Reply:
you can use an associative array of files with keys `image`, `audio`, `video` and select uploaded file accordingly.